PaloAlto/Jobs/Inventory.cs

```diff
@@ -79,25 +80,26 @@ private JobResult PerformInventory(InventoryJobConfiguration config, SubmitInven
                     config.JobHistoryId, ServerUserName, ServerPassword);
                 if (!valid) return result;
 
+                //Get the list of certificates and Trusted Roots
+                _client =
+                    new PaloAltoClient(config.CertificateStoreDetails.ClientMachine,
+                        ServerUserName, ServerPassword); //Api base URL Plus Key
+
                 _logger.LogTrace("Store Properties are Valid");
-                _logger.LogTrace($"Inventory Config {JsonConvert.SerializeObject(config)}");
+                _logger.LogTrace($"Inventory Config {_client.MaskSensitiveData(JsonConvert.SerializeObject(config))}");
                 _logger.LogTrace(
                     $"Client Machine: {config.CertificateStoreDetails.ClientMachine} ApiKey: {config.ServerPassword}");

```

Comment:
@bhillkeyfactor you masked ServerPassword in the MaskSensitiveData method, but it looks like its written into a trace log directly here

Reply:
@doebrowsk - good catch, i'll fix that.